cfme/common/vm_views.py

```diff
@@ -0,0 +1,156 @@
+# -*- coding: utf-8 -*-
```

Comment:
@izapolsk @mfalesni Thoughts on these common views? I'm still moving some from instances into here where there is overlap.

Some of the pages just have 'Instances' or 'Virtual Machine' (The class.VM_TYPE value) in the title, so I'm thinking we can define the common view and write the is_displayed property to verify against `self.context['object'].VM_TYPE`